src/js/example-app.html

```diff
@@ -6,6 +6,13 @@
   <script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js" crossorigin></script>
   <!-- this HTML snippet is appended to <<imports>> -->
   <script src="https://unpkg.com/babel-standalone@6/babel.min.js"></script>
+  <!-- this HTML snippet is appended to <<imports>>  -->
+  <script src="https://unpkg.com/@rjsf/core/dist/react-jsonschema-form.js"></script>
+  <!-- this HTML snippet is appended to <<imports>>  -->
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha384-HSMxcRTRxnN+Bdg0JdbxYKrThecOKuH5zCYotlSAcp1+c8xmyTe9GYg1l9a69psu" crossorigin="anonymous">
```

Comment:
Newest version of Bootstrap is 4.5.0. Can you update it to the latest version?

See https://www.bootstrapcdn.com/

Reply:
The react-jsonschema-form component does not support it, see https://react-jsonschema-form.readthedocs.io/en/latest/usage/themes/#supported-themes . Tried Material UI, but could not get with `<script/>` tags.